styles/components/card.css

```diff
@@ -0,0 +1,95 @@
+h2,
+p {
+  background-color: inherit;
+}
+
+.card {
+  background-color: var(--white);
+  border-radius: 2.5%;
+  overflow: hidden;
+  margin: 3rem;
+  box-shadow: 12px 12px 16px 0 var(--shadow-black),
+    -8px -8px 12px 0 var(--shadow-grey);
+}
+
+.vertical-card {
+  max-width: 20rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.horizontal-card {
+  display: flex;
+  max-width: 30rem;
+}
+
+.text-card {
+  max-width: 15rem;
+}
+.card-content {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  color: var(--black);
+}
+.card-title {
+  color: var(--primary-color);
+}
+
+.card-subtitle {
+  font-size: small;
+  margin-bottom: 0.5rem;
+  color: var(--black);
+}
+
+.card-para {
+  color: var(--grey);
+}
+
+.card .btn {
+  box-shadow: none;
+}
+
+.badge-card {
+  position: relative;
+}
+.badge {
+  background-color: var(--primary-color);
+  position: absolute;
+  color: var(--white);
+  left: -2.5rem;
+  top: 1rem;
+  transform: rotate(-45deg);
+  padding: 0.5rem 3rem;
+  height: auto;
+  width: auto;
+  border-radius: 6px;
+}
+
+.dismiss-card {
+  position: relative;
+}
+
+.dismiss {
+  position: absolute;
+  top: 0.8rem;
+  right: 0.625rem;
+}
+
+.overlay-card {
+  position: relative;
+}
+
+.overlay {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 1.5rem;
+  height: 100%;
+  width: 100%;
+  z-index: 999;
+  background-color: var(--grey);
```

Comment:
Suggestion to not use actual color value in variable naming because you won't be able to change it later inacase you need which basically defeats the purpose of having a variable.